src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -148,7 +136,7 @@ trait SharedFileSystem {
       val unwrappedMap = outputMappings collect { case (name, Success(wdlValue)) => name -> CallOutput(wdlValue, wdlValue.getHash) }
       Success(unwrappedMap.toMap)
     } else {
-      val message = taskOutputFailures collect { case (name, Failure(e)) => s"$name: $e" }
+      val message = taskOutputFailures collect { case (name, Failure(e)) => s"$name: $e\n${e.getStackTrace}" }
```

Comment:
I think you want the `ExceptionUtils` version, this will produce gibberish
```
scala> val e = new Throwable
e: Throwable = java.lang.Throwable

scala> e.getStackTrace.toString
res2: String = [Ljava.lang.StackTraceElement;@63753b6d
```